### ISSUE TITLE
zh-Hant is showing up as zh-Hans

### ISSUE THREAD

**Issue by fchicken** [OneSignal/OneSignal-Android-SDK#98]
I'm really sure that most of my users' locale is zh-TW. They should be marked as zh-Hant in the language field, but all of them are zh-Hans.

<img width="1061" alt="screen shot 2016-08-31 at 2 16 31 pm" src="https://cloud.githubusercontent.com/assets/10477535/18118142/9eac2850-6f85-11e6-9196-e0427b966077.png">

**Comment by iForests** [comment]
I can confirm this issue. All my Android users whose locale should be zh-Hant were logged as zh-Hans on OneSignal.

When I look into the source code of OneSignal's SDK, it simply returns 2-letters language code which is insufficient for distinguishing zh-Hant from zh-Hans.

Maybe you can use the country code to solve this issue?

```
String lang = Locale.getDefault().getLanguage();
String country = Locale.getDefault().getCountry();
```

|          | Traditional Chinese | Simplified Chinese |
|----------|---------------------|--------------------|
| lang | zh                  | zh                 |
| country  | TW                  | CN                 |

**Comment by jkasten2** [comment]
@fchicken @iForests 
I can add the following which should correct the issue you are seeing.
```Java
String lang = Locale.getDefault().getLanguage();
if (lang.equals("zh"))
         return lang + "-" + Locale.getDefault().getCountry();
```

Our back end will then assign `zh-TW` and `zh-HK` to `zh-Hant`. Every other `zh-*` result will be set to ` zh-Hans`. Are their any other exceptions?

**Comment by iForests** [comment]
@jkasten2 OK! Many thanks for the prompt reply.

**Comment by jkasten2** [comment]
Fixed in commit 1750f18ce5ea2bf2580341622b1e28010dba7548. It will be included in the next release.
